### DISCUSSION TITLE
Single hand analysis

### DISCUSSION THREAD

**Discussion by sievers2501** [kdblocher/bridge#32] [Ideas]
Ultimately, if this program continues to advance, we will want to decide how it will be used.  Kyle and I have had several discussions about comparing bids or systems of bids, and I think that is clearly one of the great rewards that this effort might produce.  

Nevertheless, here is one use I really see.  I was playing at a tournament the other day when I picked up KTx, KJ9xx, xxx, Qx.  RHO opened 1H, I passed, lefty passed, and partner balanced in with 2C.  The question is should I pass or bid 2N?  I would like to run 100 hands, given my partner's 2C bid definition, and see how likely 2NT gets us into trouble vs how likely pass gets us a bad result.  We need a name for this potential feature.  One hand analysis?

**Comment by kdblocher** [comment]
@sievers2501 We need to define how to pass (#39), and how to handle interference (#27) , before being able to properly deal with your example. We can certainly hook up a double dummy solver (#40) to analyze uncontested scenarios.
